mne/viz/_3d.py

```diff
@@ -305,10 +323,18 @@ def plot_head_positions(
 
         if destination is not None:
             vals = np.array(
-                [destination[:, 3], rot_to_quat(destination[:, :3])]
+                [1000 * destination[:3, 3], rot_to_quat(destination[:3, :3])]
```

Comment:
huh, was position previously in meters but labeled as millimeters? oops.

Reply:
They were in mm before but it was done by modifying `destination` inplace earlier. So no bugfix just a change in where the multiplication happens!